bucket/spotx.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "1.2",
+    "description": "Blocks ads and updates for the desktop version of Spotify and more.",
+    "homepage": "https://github.com/amd64fox/SpotX/",
+    "license": "MIT",
+    "url": "https://github.com/amd64fox/SpotX/releases/download/1.2/Install.bat",
+    "hash": "fa1c227614596f6eb930c08566a7d46af9b2590be78562cb2059bb6230cf7d4e",
+    "post_install": "cd $env:USERPROFILE\\scoop\\apps\\spotx\\1.2\\; .\\Install.bat",
+    "checkver": "github",
```

```suggestion
    "checkver": {
        "script": [
            "# Using script to get version number from date, e.g. 6 Mar, 2019 -> 2019.03.06",
            "$url1 = 'https://api.github.com/repos/amd64fox/SpotX/releases/latest'",
            "$regex1 = 'tag/([\\d.]+)'",
            "$cont = $(Invoke-WebRequest $url1).Content",
            "if(!($cont -match $regex1)) { error \"Could match '$regex1' on '$url1'\"; return }",
            "$app_ver = $matches[1]",
            "",
            "$url2 = 'https://github.com/amd64fox/SpotX/commits/main/Install.ps1'",
            "$regex2 = 'Commits on ([\\w\\s,]+)</h2>'",
            "$cont = $(Invoke-WebRequest $url2).Content",
            "if(!($cont -match $regex2)) { error \"Could match '$regex2' on '$url2'\"; return }",
            "$script_ver = $(Get-Date $matches[1]).ToString('yyyyMMdd')",
            "Write-Output \"$app_ver-$script_ver\""
        ],
        "regex": "([\\w.-]+)"
    },
    "autoupdate": {
        "url": "https://raw.githubusercontent.com/amd64fox/SpotX/main/Install.ps1"
    }
```

This **checkver script** will help us track minor changes, so that **hash errors** will not happen.